### PR TITLE
dotCMS/core#19798 add-show-starter-on-my-account-dialog

### PR DESCRIPTION
**Pull request by alfredo-dotcms**

### CODE REVIEW

File: src/app/view/components/dot-toolbar/components/dot-my-account/dot-my-account.component.spec.ts [hunk 1]
```diff
@@ -207,13 +223,35 @@ describe('DotMyAccountComponent', () => {
...
     });
 
+    it(`should call to add starter method in account service`, async () => {
+        spyOn<any>(accountService, 'addStarterPage').and.returnValue(of({ entity: {} }));
+        fixture.detectChanges();
+        comp.showStarter = true;
+        fixture.detectChanges();
+        const toggleShowStarter = de.query(By.css('#dot-my-account-show-starter'));
+        toggleShowStarter.triggerEventHandler('onChange', {});
+        expect(accountService.addStarterPage).toHaveBeenCalledTimes(1);
```
Comment (fmontes) [inline src/app/view/components/dot-toolbar/components/dot-my-account/dot-my-account.component.spec.ts:233]: You are not testing the change event you add to the html. That's is the tests not changing the `showStarter` manually.

File: src/app/view/components/dot-toolbar/components/dot-my-account/dot-my-account.component.ts [hunk 2]
```diff
@@ -80,6 +83,13 @@ export class DotMyAccountComponent implements OnInit, OnDestroy {
             }
         };
 
+        this.dotMenuService
+            .isPortletInMenu('starter')
+            .pipe(take(1))
+            .subscribe((showStarter: boolean) => {
+                this.showStarter = showStarter;
```
Comment (fmontes) [inline src/app/view/components/dot-toolbar/components/dot-my-account/dot-my-account.component.ts:90]: Perfect candidate for `async` pipe.

File: src/app/view/components/dot-toolbar/components/dot-my-account/dot-my-account.component.ts [hunk 3]
```diff
@@ -115,6 +125,14 @@ export class DotMyAccountComponent implements OnInit, OnDestroy {
         this.changePasswordOption = !this.changePasswordOption;
     }
 
+    toggleShowStarter(): void {
```
Comment (fmontes) [inline src/app/view/components/dot-toolbar/components/dot-my-account/dot-my-account.component.ts:128]: Needs doc.

**Review by fmontes — CHANGES_REQUESTED** [review]

File: src/app/view/components/dot-toolbar/components/dot-my-account/dot-my-account.component.spec.ts [hunk 4]
```diff
@@ -136,7 +136,7 @@ describe('DotMyAccountComponent', () => {
         const confirmPassword = de.nativeElement.querySelector(
             '#dot-my-account-confirm-new-password-input'
         ).parentNode;
-        const showStarter = de.nativeElement.querySelector('#dot-my-account-show-starter');
+        const showStarter = de.query(By.css('#dot-my-account-show-starter'));
```
Comment (fmontes) [inline src/app/view/components/dot-toolbar/components/dot-my-account/dot-my-account.component.spec.ts:139]: testId?

**Review by fmontes — CHANGES_REQUESTED** [review]

**Review by fmontes — APPROVED** [review]

**Review by hmoreras — APPROVED** [review]

**Pull request merged**